.github/workflows/main.yml

```diff
@@ -232,6 +233,30 @@ jobs:
             echo "WARN: No instance archive found for simple-transfer-${GITHUB_RUN_NUMBER}.tar.gz.gpg" >&2
           fi
 
+  quickstart:
+    name: Run the Go Quickstart
+    # - always() allows evaluating further conditional expressions even if
```

Comment:
We can drop `if: always()` because the job shouldn't run if linux-build doesn't succeed since it downloads the build artifact from that job that it `needs`.

Reply:
Sent a twig PR to reflect my proposed changes: https://github.com/openziti/ziti/pull/1300